packages/itwin/tree-widget/src/test/trees/models-tree/internal/VisibilityValidation.ts

```diff
@@ -7,6 +7,7 @@ import { assert, expect } from "chai";
 import { expand, from, mergeMap } from "rxjs";
 import { PerModelCategoryVisibility } from "@itwin/core-frontend";
 import { HierarchyNode } from "@itwin/presentation-hierarchies";
+import { waitFor } from "@testing-library/react";
```

Comment:
Can you import from `TestUtils.tsx` so that we have single place where it's imported in tests.